src/connector/src/source/mqtt/source/reader.rs

```diff
@@ -29,11 +28,6 @@ use crate::source::{BoxSourceChunkStream, Column, SourceContextRef, SourceMessag
 
 pub struct MqttSplitReader {
     eventloop: rumqttc::v5::EventLoop,
-    client: rumqttc::v5::AsyncClient,
-    qos: QoS,
-    splits: Vec<MqttSplit>,
-    #[expect(dead_code)]
-    properties: MqttProperties,
```

Comment:
let's keep the props for future use (maybe), just allow dead code here should be ok.